lib/src/chrome.dart

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Not sure how this is done on the Dart team, but the general policy we use for Flutter is to use a single year for all files (in our case the earliest date we had in any of them, which was 2013). We used to use a policy that new files got the current year, old files preserved their headers, but that changed a few years back.

If current-year is still the right approach for Dart packages, then LGTM. Otherwise we should probably (in a separate PR), set these all consistently to 2013 in this repo, which is the earliest date I see in our headers.

Reply:
Most other packages I touch follow the current-year pattern.